packages/jaeger-ui/src/components/TracePage/CriticalPath/index.test.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TraceCriticalPath from './index';
+import transformTraceData from '../../../model/transform-trace-data';
+import criticalPathSections from './testResults/test1';
+
+const testTrace = require('../TraceStatistics/tableValuesTestTrace/testTraceNormal.json');
+
+const transformedTrace = transformTraceData(testTrace);
+
+const defaultProps = {
+  trace: transformedTrace,
+};
+
+describe('<TraceCriticalPath />', () => {
+  it('Critical path sections', () => {
+    const consoleLogMock = jest.spyOn(console, 'log').mockImplementation();
+    TraceCriticalPath(defaultProps);
+    expect(consoleLogMock).toHaveBeenCalledWith(criticalPathSections);
```

Comment:
We should assert the structure of data which is returned, and ensure that the values returned are the one that we are expecting. Please add more assertions in the test-case

Reply:
I added required assertions , are anymore required

Reply:
So far, we have included syntactical assertions, which is good, as it helps us make sure that we don't break the structure of data we use.

However, we haven't included unit-tests for logical changes. For certain cases, where you know which spans are included and excluded, let's assert the logical expected values. For e.g. you might have some spans which **should/should not** be included in the calculation of the critical path. Make sure that we also include the expectation while writing unit-tests.

Reply:
Can you resolve the issue, if it's addressed?